treeherder/config/settings.py

```diff
@@ -585,4 +585,10 @@
 # Configuration for elasticsearch backend
 ELASTICSEARCH_URL = env.str('ELASTICSEARCH_URL', default='')
 
-TRUNK_REPO_NAMES = ['mozilla-central', 'mozilla-inbound', 'autoland', 'fx-team']
+REPO_GROUPS = {
+    'trunk': ('mozilla-central', 'mozilla-inbound', 'autoland'),
```

Comment:
In the "not an alias" case, the `name__in` is passed a list (ie: `param = [name]`), whereas these are tuples. I think it would be good for them to either be consistently tuples or lists -- can't quite decide which, possibly 60:40 to lists, but up to you.

Reply:
I'll change them all to lists. IIRC, putting a single string in parenthesis throws an error, so that's why I used a list there.

Reply:
oh - that is weird! Thanks for pointing that out.